.gitlab/packaging/oci.yml

```diff
@@ -101,11 +128,44 @@ installer_oci:
   variables:
     OCI_PRODUCT: "datadog-installer"
 
+ddot_win_oci:
+  extends: .package_oci
+  stage: packaging
+  needs:
+    - job: "windows_zip_ddot_x64"
+      artifacts: true
+  rules:
+    - !reference [.except_mergequeue]
+    - when: on_success
+  script:
```

Comment:
Instead of repeating all this logic, perhaps we could better either:
- Make `.package_oci`'s logic more generic such that we can reuse its `script` as is, or
- Extract common logic into a separate script and reuse that.